nodejs-jquery-ujs/nodejs-jquery-ujs.spec

```diff
@@ -0,0 +1,57 @@
+%global npm_name jquery-ujs
+
+Name: nodejs-%{npm_name}
+Version: 1.2.1
+Release: 1%{?dist}
+Summary: Ruby on Rails unobtrusive scripting adapter for jQuery, for npm
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/shakacode/jquery-ujs.git
+Source0: http://registry.npmjs.org/jquery-ujs/-/jquery-ujs-1.2.1.tgz
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+BuildRequires: nodejs-devel
+BuildRequires: nodejs-packaging
+Requires: npm(jquery) >= 1.8.0
+BuildArch: noarch
+ExclusiveArch: %{nodejs_arches} noarch
+Provides: npm(%{npm_name}) = %{version}
+
+%description
+Ruby on Rails unobtrusive scripting adapter for jQuery, for npm
+
+%package doc
+Summary: Documentation for nodejs-%{npm_name}
+Group: Documentation
+Requires: nodejs-%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+This package contains documentation for nodejs-%{pkg_name}
+
+%prep
+%setup -q -n package
+
+%build
+#nothing to do
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr script src test MIT-LICENSE *.json *.md %{buildroot}%{nodejs_sitelib}/%{npm_name}
+
+%clean
+rm -rf %{buildroot}
+
+%files
+%{nodejs_sitelib}/%{npm_name}
+%exclude %{nodejs_sitelib}/%{npm_name}/script
+%exclude %{nodejs_sitelib}/%{npm_name}/*.json
+%exclude %{nodejs_sitelib}/%{npm_name}/*.md
+%exclude %{nodejs_sitelib}/%{npm_name}/.*
+
+%files doc
+%doc %{nodejs_sitelib}/%{npm_name}/README.md
+%doc %{nodejs_sitelib}/%{npm_name}/CONTRIBUTING.md
+%doc %{nodejs_sitelib}/%{npm_name}/MIT-LICENSE
```

Comment:
Should be in the main subpackage's %files list instead, so it's always installed.